java/bed-allocation/README.adoc

```diff
@@ -49,6 +49,79 @@ Then try _live coding_:
 
 Notice that those changes are immediately in effect.
 
+[[enterprise]]
+== Run the application with Timefold Solver Enterprise Edition
+
+For high-scalability use cases, switch to https://docs.timefold.ai/timefold-solver/latest/enterprise-edition/enterprise-edition[Timefold Solver Enterprise Edition],
+our commercial offering.
+https://timefold.ai/contact[Contact Timefold] to obtain the credentials required to access our private Enterprise Maven repository.
+
+. Create `.m2/settings.xml` in your home directory with the following content:
```

Comment:
This is a Maven-specific instruction, it belongs in the Maven tab.
Gradle does this differently.